src/index.html

```diff
@@ -9,6 +9,39 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a class="logo" href="#">
+        <img src="images/logo.png" alt="logo">
+      </a>
+      <nav class="menu">
+        <ul class="menu-list">
+          <li class="menu-item">
```

Comment:
Check the code style rules, please
![image](https://github.com/mate-academy/layout_moyo-header/assets/46566190/0af06b78-0e3c-4820-8bca-437abfeff2cb)